ext/ftp/ftp.stub.php

```diff
@@ -57,63 +57,63 @@ function ftp_systype($ftp): string|false {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_fget($ftp, $fp, string $remote_file, int $mode = FTP_BINARY, int $resumepos = 0): bool {}
+function ftp_fget($ftp, $stream, string $remote_filename, int $mode = FTP_BINARY, int $resume_position = 0): bool {}
```

Comment:
Maybe `$fp` -> `$target` and `$remote_file` -> `$source` could work here too. From the types and context it is clear what is expected.

While `$resumepos` is not a word, `resume_position` is a bit long. Maybe `$offset` could work?

Reply:
Hmm, `$offset` seems good for me. @nikic ?